### ISSUE TITLE
try building more libraries in parallel

### ISSUE THREAD

**Issue by DanGrayson** [Macaulay2/M2#1265]

**Comment by mahrud** [comment]
Is this a duplicate of #116?

**Comment by DanGrayson** [comment]
We might as well just close #116 and keep this.  I should mention that right now most of the libraries are built with `make -j1`, probably needlessly.  Fixing that would be simple, simpler than the discussion in #116 went.
